docs/pages/workflow/logging.md

```diff
@@ -20,28 +23,33 @@ While it's usually not necessary, if you want to see logs for everything happeni
 
 The following instructions apply to macOS.
 
-- `brew install --HEAD libimobiledevice -g`
-- Plug your phone in
-- `idevicepair pair`
-- Press accept on your device
-- Run `idevicesyslog`
+- Install `libimobiledevice`:
+  <Terminal cmd={["$ brew install --HEAD libimobiledevice -g"]} />
+- Plug your phone in and run:
+  <Terminal cmd={["$ idevicepair pair"]} />
+- Press accept on your device, after that run:
+  <Terminal cmd={["$ idevicesyslog"]} />
 
 ### View logs for an iOS simulator
 
 #### Option 1: Use GUI log
 
-- In simulator, press <kbd>Cmd ⌘</kbd> + <kbd>/</kbd>, _or_ go to `Debug -> Open System Log` -- both of these open a log window that displays all of the logs from your device, including the logs from your Expo app.
+- In simulator, press <kbd>Cmd ⌘</kbd> + <kbd>/</kbd>, _or_ go to `Debug -> Open System Log`.<br />Both of these open a log window that displays all of the logs from your device, including the logs from your Expo app.
```

Comment:
```suggestion
- In simulator, press <kbd>Cmd ⌘</kbd> + <kbd>/</kbd>, or go to `Debug -> Open System Log`.<br />Both of these open a log window that displays all of the logs from your device, including the logs from your Expo app.
```

I don't think italicizing the word "or" has any significance. (I know it was there before and was not added by your commit).